sensor_msgs/LaserScan laser_scan
---
Circle[] circles